src/core/pipelines/manifest/create_manifest_pipeline.ts

```diff
@@ -166,7 +169,8 @@ export default function createManifestPipeline(
           for (let i = 0; i < warnings.length; i++) {
             warning$.next(warnings[i]); // TODO not through warning$
           }
-          return { manifest, sendingTime };
+          const parsingTime = performance.now() - parsingTimeStart;
```

Comment:
Why not measuring the parsing time before the for loop ?

Reply:
Good question. I considered it to be part of the parsing time.

Warning events (and corresponding event handlers) still concern the parsing phase, no?